cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -10,38 +10,43 @@
 
    value:                              Object defined from a StreamField block.
 
-   value.half_width:                   A Boolean indicating whether the
+   value.is_half_width:                A boolean indicating whether the
                                        list items should be at half width.
                                        Defaults to False.
 
-   value.slug:                         TODO: Add data type and description.
+   value.slug:                         A string for the header on all
+                                       related metadata items.
 
    value.content:
```

Comment:
Add a description of this, too? "A list of metadata blocks" or something ought to do it.